src/goto-programs/remove_returns.cpp

```diff
@@ -13,17 +13,24 @@ Date:   September 2009
 
 #include "remove_returns.h"
 
+#include <util/c_types.h>
+#include <util/exception_utils.h>
+#include <util/fresh_symbol.h>
 #include <util/std_expr.h>
 
 #include "goto_model.h"
 
 #include "remove_skip.h"
 
+#include "remove_function_pointers.h"
+
 class remove_returnst
 {
 public:
-  explicit remove_returnst(symbol_table_baset &_symbol_table):
-    symbol_table(_symbol_table)
+  explicit remove_returnst(
```

Comment:
Are those changes co-ordinated with @kroening? If it's not the case, please do get together to make sure the two of you don't end up working in opposite directions.